hasher-matcher-actioner/.devcontainer/startup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -e
 export OMM_CONFIG=/workspace/reference_omm_configs/development_omm_config.py
-MIGRATION_COMMAND=1 flask --app OpenMediaMatch.app db upgrade --directory /workspace/src/OpenMediaMatch/migrations
-flask --app OpenMediaMatch.app run --debug
+MIGRATION_COMMAND=1 flask --app src/OpenMediaMatch.app db upgrade --directory /workspace/src/OpenMediaMatch/migrations
```

Comment:
@juanmrad  - can you double check that this is correct? This script should be running with a working directory

Reply:
Yes it should be running on the working directory. Where you having issues when running the shell script outside of dev container? or where you running it inside?

Reply:
hey @juanmrad this was failing on the dev container startup
```
Running the postAttachCommand from devcontainer.json...

[4812 ms] Start: Run in container: /bin/sh -c /workspace/.devcontainer/startup.sh
Error: Could not import 'OpenMediaMatch.app'.
```
this is after opening vscode in the `hasher-matcher-actioner` dir and selecting "Reopen in Container"
<img width="603" alt="image" src="https://github.com/user-attachments/assets/193aa9c6-2d29-4f44-ac92-8559e13d5784">

Reply:
Seems like I was able to repro on a whole new build 🤔. Not sure how this may have changed to cause the problem.

Reply:
Seems like Juan is convinced, lets land it. I'll re-test on windows just to make sure there's not any evil differences between different vscodes.